### DISCUSSION TITLE
Button and Relay pins of WLAN power plug

### DISCUSSION THREAD

**Discussion by kahlenberg** [arendst/Tasmota#21943] [Q&A] [answered]
Hi,
I have a Delock power plug (https://www.delock.de/produkt/11826/merkmale.html) and I would like to know which pins are used for button, LED and relay. Where in source code can I find it?
Thanks.

**Comment by sfromis** [comment, marked as answer]
Not in any source code (unless maybe if you have hacked Delock to get their source code), but the Tasmota template config is here: https://templates.blakadder.com/delock_11826.html
(Assuming that this is the same device)
